tests/k8s/1-045_validate_applicationset_image_override/01-add_applicationset_image.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: basic
+spec:
```

Comment:
Can you add a comment with what is expected here? Which image value will have high priority.